skill/file-structure.md

```
@@ -0,0 +1,223 @@
+| `currentStep` | number? | 0-based index of currently executing step; null if none |
+
+---
+
+## `artifact-index.jsonl` — Artifact Index
+
+Location:
+- Run-level: `.kata/runs/<run-id>/artifact-index.jsonl`
+- Flavor-level: `.kata/runs/<run-id>/stages/<category>/flavors/<name>/artifact-index.jsonl`
+
+Each line is a JSON object:
+```json
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)